Can't build Android version

When run `npm run build:android-release`, exit with error:
```shell
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

* Get more help at https://help.gradle.org

BUILD FAILED in 48s
 FAILED
28 actionable tasks: 28 executed
/home/zhong/wallet/platforms/android/gradlew: Command failed with exit code 1 Error output:
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
/home/zhong/wallet/platforms/android/app/src/main/java/com/gae/scaffolder/plugin/MyFirebaseInstanceIDService.java:6: error: cannot find symbol
import com.google.firebase.iid.FirebaseInstanceIdService;
                              ^
  symbol:   class FirebaseInstanceIdService
  location: package com.google.firebase.iid
/home/zhong/wallet/platforms/android/app/src/main/java/com/gae/scaffolder/plugin/MyFirebaseInstanceIDService.java:11: error: cannot find symbol
public class MyFirebaseInstanceIDService extends FirebaseInstanceIdService {
                                                 ^
  symbol: class FirebaseInstanceIdService
/home/zhong/wallet/platforms/android/app/src/main/java/com/gae/scaffolder/plugin/MyFirebaseInstanceIDService.java:15: error: method does not override or implement a method from a supertype
    @Override
    ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
3 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:compileReleaseJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

```

--- a/app-template/config-template.xml
+++ b/app-template/config-template.xml
@@ -85,7 +85,7 @@
       <variable name="SECOND_URL_SCHEME" value="*APPURI*" />
       <variable name="THIRD_URL_SCHEME" value="bitcoincash" />
     </plugin>
-    <plugin name="cordova-plugin-fcm" spec="https://github.com/cmgustavo/cordova-plugin-fcm.git#v4.1" />
+    <plugin name="cordova-plugin-fcm-ng" spec="3.0.1" />
 
     <plugin name="cordova-plugin-wkwebview-engine" spec="1.1.4" />
 

--- a/app-template/config-template.xml
+++ b/app-template/config-template.xml
@@ -85,7 +85,7 @@
       <variable name="SECOND_URL_SCHEME" value="*APPURI*" />
       <variable name="THIRD_URL_SCHEME" value="bitcoincash" />
     </plugin>
-    <plugin name="cordova-plugin-fcm-ng" spec="3.0.1" />
+    <plugin name="cordova-plugin-fcm-ng" spec="3.0.2" />
 
     <plugin name="cordova-plugin-wkwebview-engine" spec="1.1.4" />
 

--- a/package.json
+++ b/package.json
@@ -126,7 +126,7 @@
     "cordova-plugin-customurlscheme-ng": "5.0.0",
     "cordova-plugin-device": "2.0.1",
     "cordova-plugin-dialogs": "2.0.1",
-    "cordova-plugin-fcm-ng": "3.0.1",
+    "cordova-plugin-fcm-ng": "3.0.2",
     "cordova-plugin-file": "6.0.1",
     "cordova-plugin-globalization": "1.0.9",
     "cordova-plugin-inappbrowser": "2.0.2",

--- a/package.json
+++ b/package.json
@@ -126,7 +126,7 @@
     "cordova-plugin-customurlscheme-ng": "5.0.0",
     "cordova-plugin-device": "2.0.1",
     "cordova-plugin-dialogs": "2.0.1",
-    "cordova-plugin-fcm": "https://github.com/cmgustavo/cordova-plugin-fcm.git#v4.1",
+    "cordova-plugin-fcm-ng": "3.0.1",
     "cordova-plugin-file": "6.0.1",
     "cordova-plugin-globalization": "1.0.9",
     "cordova-plugin-inappbrowser": "2.0.2",
@@ -237,7 +237,7 @@
         "ANDROID_HOST": " ",
         "ANDROID_PATHPREFIX": "/"
       },
-      "cordova-plugin-fcm": {},
+      "cordova-plugin-fcm-ng": {},
       "cordova-plugin-globalization": {},
       "cordova-plugin-inappbrowser": {},
       "cordova-plugin-screen-orientation": {},

--- a/src/providers/push-notifications/push-notifications.ts
+++ b/src/providers/push-notifications/push-notifications.ts
@@ -67,13 +67,6 @@ export class PushNotificationsProvider {
 
   public handlePushNotifications(): void {
     if (this.usePushNotifications) {
-      this.FCMPlugin.onTokenRefresh().subscribe(token => {
-        if (!this._token) return;
-        this.logger.debug('Refresh and update token for push notifications...');
-        this._token = token;
-        this.enable();
-      });
-
       this.FCMPlugin.onNotification().subscribe(async data => {
         if (!this._token) return;
         this.logger.debug(

